src/schemas/space.json

```diff
@@ -234,8 +234,31 @@
           "required": ["name"],
           "additionalProperties": false
         },
-        "delegation": {
```

Comment:
We shouldn't remove this, may break some stuff

Reply:
It's not used anywhere and it's not a required field. How can I break anything?

Reply:
Yea `delegation` was added before when we thought of refactoring delegation but we didn't work on it anymore, I think it should be fine, couldn't find any references with this, not even on hub